api/idpay.py

```diff
@@ -636,3 +637,15 @@ def get_initiative_info(selfcare_token: str,
             'Content-Type': 'application/json',
         },
         timeout=settings.default_timeout)
+
+
+def put_minint_associate_user_and_payment(fiscal_code: str,
+                                          transaction_id: str):
+    response = requests.put(
+        f'{settings.base_path.IO}{settings.IDPAY.domain}{settings.IDPAY.MININT.domain}{settings.IDPAY.endpoints.payment.path}/{transaction_id}/user',
+        headers={
+            settings.API_KEY_HEADER: secrets.api_key.IDPAY_MIN_INT_PRODUCT,
+            'Fiscal-Code': f'{fiscal_code}'
```

Comment:
Probably string format can be avoided here:
```suggestion
            'Fiscal-Code': fiscal_code
```